Seq of play improvements (#2)
raunc/eutpop-ref-sheet: src/sequence_of_play.tex

	\subsubsection*{\zsavepos{arrowNprInvasionsEnd}NPR Invasions \normal{(p.~36)}}
	\begin{itemize}
		\item Resolve NPRs in alphabetical order
		\item \strong{Invasion takes place if}
		\begin{itemize}
			\item The NPR is not an Active Ally, \conj{and}
			\item There are no Hostile Units in Areas where the NPR Controls Prov., \conj{and}
			\item There is at least one eligible target Area Adjacent to the NPR
		\end{itemize}
		\item An \strong{Area is eligible if}
		\begin{itemize}
			\item It contains Hostile \towns/\vassals, \conj{and}
			\item If Adj. by Sea only, a connecting Sea Zone contains no Hostile Ships, \conj{and}
			\item It contains no non-Hostile PR Units
		\end{itemize}
	\end{itemize}
	\subsubsection*{Invasion sequence}
	\begin{itemize}
		\item Pick an eligible Area. Priority:
		\begin{enumerate}
			\item NPR's Capital Area
			\item Area with the most NPR's Core Prov.
			\item Area Adjacent by Land
			\item Random
		\end{enumerate}
		\item Add \unrest to 2 Enemy \towns/\vassals in the target Area (pick \az)
		\item Place NPR Units equal to ½ of Tax Value currently Controlled by the NPR (including Vassals)

> 		\item NPRs invade at normal strength even if not enough tokens in Supply

Reviewer: This is stated on page 2 instead of 36 like other things in this section. Please add a reference

		\item Resolve Battle immediately (if any)
		\begin{itemize}
			\botrule{\item Place Bot Army in Area (p.~2)}
		\end{itemize}
		\item Leave the NPR Units on board as \rebels
	\end{itemize}
\end{eubox}\end{dynamiccontents*}

\begin{dynamiccontents*}{framePeaceSeq}\begin{eubox}{\fhPeaceSeq}
	\begin{multicols}{3}
		\subsubsection*{\zsavepos{arrowPeaceSeqEnd}Peace Resolution \normal{(p.~29-30)}}
		\begin{itemize}
			\item While \strong{Def. the HRE} is active (p.~44)
			\begin{itemize}
				\item PRs at War with Emperor may enforce Peace on Imp. Subjects only if Victory over Emp. is achieved
				\item Emp. negotiates on behalf of NPR Imperial Subjects as for Active Allies
			\end{itemize}
			\item Peace is resolved with 1 Enemy at a time, except Active Allies, in any order
			\item Treat Vassals as part of their Overlord
			\item In each step, resolve Peace in Turn order
			\item Victory is only assigned to PR
			\begin{itemize}
				\item who achieved Total/Partial Victory, \conj{or}
				\item whose Enemy Surrendered, \conj{or}

> 				\item if agreed as such in Negotiated Peace

Reviewer: ```suggestion
				\item Who achieved Total/Partial Victory, \conj{or}
				\item Whose Enemy Surrendered, \conj{or}
				\item If agreed as such in Negotiated Peace
```
Let's capitalize first word of each line (same has been done elsewhere in this document)

			\begin{itemize}
				\item If Enemy is a PR, your Deployed Land Units must outnumber Enemy 2:1
				\item If Enemy is NPR or Bot, your Depl. Land Units must outnum. \rebels in your Areas plus Enemy Depl. Land Units
				\item Units on Dist. Continents where the other Realm has no \towns do no count
			\end{itemize}
			\item If PR Enemy has \strong{Active Allies}
			\begin{itemize}
				\item If and only if achieved against PR, then achieved against their Active Allies
				\item Only a single Peace Term may be chosen for the PR and their Act. Allies
			\end{itemize}
			\item If \strong{inconclusive}
			\begin{itemize}
				\item NPR accepts White Peace
				\item Bot accepts White Peace if Tax Value of their \towns Occupied by you > your \towns Occupied by them
			\end{itemize}
		\end{itemize}

		\subsubsection*{4. Aftermath}
		\begin{itemize}
			\item Raise \authority by 1 (p.~44-45)
			\begin{itemize}
				\item if PR Emp. wins against ext. Aggressor \conj{and} no HRE Prov. is ceded
				\item for each Area joining HRE
			\end{itemize}
			\item Drop \authority by 1 (p.~44-45)
			\begin{itemize}
				\item if PR Emp. loses against ext. Aggressor

> 				\item for each Area leaving HRE

Reviewer: ```suggestion
				\item If PR Emp. wins against ext. Aggressor \conj{and} no HRE Prov. is ceded
				\item For each Area joining HRE
			\end{itemize}
			\item Drop \authority by 1 (p.~44-45)
			\begin{itemize}
				\item If PR Emp. loses against ext. Aggressor
				\item For each Area leaving HRE
```
First word of the line is capitalized in other places. Let's do the same here

> 				\item for each Area leaving HRE

Reviewer: I like that you managed to remove the separate box that just contained notes about Imperial Authority and Truces, but I think that Aftermath section is not a good place for them.

As the rules do not specify the exact moment where we should check for Authority raise/drop, I'd expect it to happen immediately after conditions are met. In case of peace resolution, it would mean right after concluding a peace that involves HRE. So, I'd move it out of Aftermath section, either to the first part of this box (somewhere before automatic white peace), or at the very end, adding another heading, possibly separated by a horizontal line

Author: In the text column under "D. Peace Resolution" would be a nice place, allowing also a better space usage possibly.

			\end{itemize}
			\item PRs at Peace must move Units to the nearest Friendly Area (count 1 for each Area/Sea Zone)
			\begin{itemize}
				\item If equally close, then PR may choose
				\item This movement has no cost
			\end{itemize}
			\item PRs at Peace flip their \activeallies back to \alliance
			\item PRs at Peace remove Allied Units

> 			\item PRs at Peace flip their War/Truce tokens

Reviewer: PRs who are not at Peace might need to flip these tokens too, if they have made Peace with just some of their Enemies. So perhaps it would be better to word it something like "PRs flip their War/Truce tokens when Peace is made".

Also see the comment about Imperial Authority. I think the same applies here and better to move it out of Aftermath section

Author: Same as before, under "D. Peace Resolution" would be a solution.

		\end{itemize}
	\end{multicols}
\end{eubox}\end{dynamiccontents*}

\sectionbg{Sequence \normal{\Large{(p.~8-11)}}}
\subsection*{1. Draw Cards}
\begin{itemize}
	\item May not score Missions in Phase 1 (p.~42)
\end{itemize}

> \subsubsection*{A. Reveal Events}
> \begin{itemize}
> 	\item Draw 1 + \# PRs events, reveal 3 of them
> \end{itemize}
> \smallheading{Optional Rule 7: Revealing More \normal{(p.~39)}}
> \begin{itemize}
> 	\item If 6 PRs, reveal 4 of them
> \end{itemize}

Reviewer: ```suggestion
\subsubsection*{A. Reveal Events}
\begin{itemize}
	\item Draw \# of PRs + 1, reveal 3 of them
\end{itemize}
\smallheading{Optional Rule 7: Revealing More \normal{(p.~39)}}
\begin{itemize}
	\item If 6 PRs, reveal 4 Events
\end{itemize}
```
What do you think about this small wording change here?

		\end{itemize}
	\end{multicols}
\end{eubox}\end{dynamiccontents*}

\sectionbg{Sequence \normal{\Large{(p.~8-11)}}}
\subsection*{1. Draw Cards}
\begin{itemize}

> 	\botrule{\item Bots ignore this Phase (p.~2)}

Reviewer: I think it is more important to have "May not score Missions in Phase 1 (p.~42)" statement here. Nothing here is applicable to bots anyway, so it doesn't seem very important to state that Bots ignore it, but the fact that missions may not be scored, might be forgotten easily.

Reviewer: Could state both, but seems there is not enough room

\end{itemize}

\subsubsection*{C. Rebels Siege \& Move}
\begin{itemize}
	\item Resolve by Area: 1. most \rebels; 2. \az
	\item In \strong{Areas with \unrest}, Rebels Siege \unrest. Priority:
	\begin{enumerate}
		\item Large Provinces
		\item If 2+ PRs with \unrest in the Area, PR with lowest Tax Income (no tie) in the Area chooses (p.~37)
		\item \az
	\end{enumerate}
	\item In \strong{Areas with no \unrest}
	\begin{itemize}
		\item If an Area with \unrest is Adj. by Land \conj{and} \rebels will not be outnum. there by PRs
		\begin{itemize}
			\item Move all but 1 \rebel. Priority:\\ 1. most \unrest; 2. most PR \towns; 3. \az
			\item Resolve Battle immediately
		\end{itemize}
		\item Otherwise remove 1\rebel
	\end{itemize}
\end{itemize}

\subsubsection*{\zsavepos{arrowPeaceSeqStart}D. Peace Resolutions\zsavepos{arrowPeaceTermsStart}}
\begin{itemize}
	\item When Emperor makes peace with an External Aggressor (p.~44-45)
	\begin{itemize}
		\item If Emp. won \conj{and} no HRE Prov. is lost
		\begin{itemize}
			\item Raise \authority by \# of Areas joining HRE
		\end{itemize}
		\item If Emp. lost
		\begin{itemize}
			\item Drop \authority by \# of Areas leaving HRE
		\end{itemize}

> 	\end{itemize}

Reviewer: ```suggestion
	\item When Emperor makes peace with an External Aggressor (p.~44-45)
	\begin{itemize}
		\item If Emp. won \conj{and} no HRE Prov. is ceded, raise \authority by 1
		\item If Emp. lost, drop \authority by 1
	\end{itemize}
	\item Raise/drop \authority by \# of Areas joining/leaving HRE (p.~45)
```
Emperor's victory/defeat raises/drops Authority by 1. Authority change for Areas joining/leaving is in addition to this and happens even when Emperor is not involved in the Peace deal. For example, it can happen when Emperor did not accept CtA from a Subject.

For this reason, I'd change the current statement to only state raise/drop by 1, and add a separate root-level point for raise/drop by number of Areas.

Didn't try out if it fits here, though. If you can't fit it here easily, please let me know and I'll try it too

Author: I modified the Truce sentence to make it fit, let me know it this is understandable.
Otherwise, an alternative solution would be to squeeze the Peace Sequence frame to 29.2 base line skips, which is not so nice but might be acceptable.

Reviewer: Not sure if "resp." would be understandable. Tried a few more alternative wordings and all the following ones fit on 1 line:

- Flip War token(s) when Peace is made
- Making Peace flips War token(s) to Truce
- Making Peace flips respective War token(s)

Do you like any of these?

Reviewer: Also noticed now that "Peace" is not capitalized in "When Emperor makes peace...". This is a game term that is always capitalized in the rulebook. Let's do the same here

Author: Great suggestions, done

> 	\item PRs making peace flip the relative War/Truce tokens

Reviewer: ```suggestion
	\item PRs making peace flip respective War/Truce tokens
```

	\drawArrow{(\nprStartX, \nprStartY) -- (\nprTurnX,\nprStartY) -- (\nprTurnX,\nprEndY) -- (\nprEndX,\nprEndY)}

	\end{tikzpicture}
\end{dynamiccontents*}

\framebreak

\subsubsection*{E. Prestige Penalties}
\begin{itemize}
	\item Flip all \disputedsuccession back to \marriage
	\begin{itemize}
		\item Lose \p3 for each unless
		\begin{itemize}
			\item Only you have \alliance with them, \conj{and}
			\item If it is an NPR, you have the most \influence (min 2) in that Realm
		\end{itemize}
	\end{itemize}
	\item Lose \prestige = Tax Val. of Occup. \towns (max 5)
\end{itemize}

\subsubsection*{F. Interregnums}
\begin{itemize}
	\item Lose 1\stability
	\item Flip \marriages on your Realm
\end{itemize}

\subsubsection*{G. Religious Dissent}
\begin{itemize}
	\item In Areas following a different Religion from your State Religion
	\begin{itemize}
		\item In \conj{each} Area containing your \towns
		\begin{itemize}
			\item Add 1\unrest to any \town of yours
		\end{itemize}
		\item If \conj{any} Area contains your \vassals
		\begin{itemize}
			\item Add 1\unrest to any \vassal of yours, \conj{or}
			\item Remove 1\influence in an affected Area

> 		\end{itemize}

Reviewer: ```suggestion
		\item In \conj{1} Area containing your \vassals
		\begin{itemize}
			\item Add 1\unrest to any \vassal of yours, \conj{or}
			\item Remove 1\influence
		\end{itemize}
```
Seemed possible to interpret the current wording so that player may add unrest to any vassal, but should be only in the Area affected by Religious Dissent. I think saying "In 1 Area containing your vassal" fixes this, and then the "in an affected Area" is redundant in the last point

		\end{itemize}
	\end{multicols}
\end{eubox}\end{dynamiccontents*}

\sectionbg{Sequence \normal{\Large{(p.~8-11)}}}
\subsection*{1. Draw Cards}
\begin{itemize}
	\item May not score Missions in Phase 1 (p.~42)
	\botrule{\item Bots ignore this Phase (p.~2)}
\end{itemize}
\subsubsection*{A. Reveal Events}
\begin{itemize}
	\item Draw 1 + \# PRs events, reveal 3 of them
\end{itemize}

> \smallheading{Optional Rule 7: Revealing More (p.~39)}

Reviewer: ```suggestion
\smallheading{Optional Rule 7: Revealing More \normal{(p.~39)}}
```
Page reference is not bold in other places, let's do the same here

	\item Max 2 in hand
\end{itemize}

\subsection*{2. Actions}
\begin{itemize}
	\item On each Turn, must take 1 Action, \conj{and}
	\item May take any \# of Minor Actions
	\botrule{\item Bots use Main Turn Structure (p.~7)}
\end{itemize}
\subsubsection*{Passing}
\begin{itemize}
	\item Allowed only if \action{Event} taken
	\item May take Minor Actions on same Turn
	\item First 2-4 PRs to Pass gain \ducats
	\item If no more PRs eligible to gain \ducats,\\ \strong{End of Action Phase} is triggered
	\begin{itemize}
		\item Each remaining PR may take 1 more Action of their choice, \conj{then}
		\item PRs who still have not taken \action{Event},\\ get one more Turn to take \action{Event}
		\item If PR has taken their final Turn, they are considered to have Passed
	\end{itemize}
\end{itemize}

\subsection*{3. Peace \& Rebels}
\subsubsection*{A. Remove Casus Belli \& Truces}
\begin{itemize}
	\item Lose \p2 if not at war with the CB target
\end{itemize}

\subsubsection*{B. NPR Invasions \normal{(advanced rules)}}
\begin{itemize}
	\item \hspacezero\zsavepos{arrowNprInvasionsStart}NPRs at War might invade PRs
	\botrules

> 	\item Bots suffer max 1 NPR invasion. Priority: 1. most \rebels; 2. \az (p.~2)

Reviewer: ```suggestion
	\item Bots suffer max 1 NPR invasion. Priority: 1. most NPR Units; 2. \az (p.~2)
```
I'd use "NPR Units" instead of Rebel Units icon because at this moment they are not considered rebels yet and the rules also say "NPR Units". Probably no practical difference and not very important, though, so feel free to ignore if you disagree

			\item Add \unrest to all Provinces gained in Areas where the Victor Occupied no Provinces
			\item Remove all Loser's \alliances/\marriages
			\item Victor may discard \claim from Area(s) of gained \vassalprovinces to remove 2 \unrest from same Area(s)

> 			\item Remove War token from the Loser

Reviewer: ```suggestion
			\item Remove War/Truce tokens from the Loser
```
There might be Truces from earlier wars

				\end{itemize}
			}
			\item Remove all Loser's \alliances/\marriages

> 			\item If Loser is Active Ally, then lose \p2

Reviewer: ```suggestion
			\item If Loser is Active Ally, the Allied PR loses \p2
```
I'd keep it more explicit, who loses prestige. If it doesn't fit, then maybe the sentence below about Authority can be shortened more. If still not enough room, then feel free to ignore for now and I'll figure something out later

\begin{tabularx}{\columnwidth}{ | X | X | }
\hline

> \rowcolor{tblbgTax} + Base and Vassal Tax Income & Do not count Occupied \towns (p.~28) \\ \hline

Reviewer: Occupied Towns count for the Occupier, so I'd say this statement is not entirely accurate. I guess it can be here if there is enough room to make it clearer that they do not count only for the Lawful Owner. But the whole statement doesn't seem important, because the Lawful Owner should have their respective town slots covered by cubes anyway to mark reduced Income

Author: Right, can mention \cube explicitly, or just think it is clear enough.

Reviewer: Yes, good idea, seems like "Do not count slots with \cubes" would fit there

\begin{tabularx}{\columnwidth}{ | X | X | }
\hline
\rowcolor{tblbgTax} + Base and Vassal Tax Income & Do not count slots with \cubes (p.~28) \\ \hline
\rowcolor{tblbgCost} - Advisor Upkeep & Papal Controller pays 1\ducat less per Advisor \\ \hline
\rowcolor{tblbgCost} - Military Maintenance & {
1\ducat per Regular Unit \newline
2\ducats per Mercenary \newline
½\ducat per Ship at sea
} \\ \hline

> \rowcolor{tblbgCost} - Plague & ½\ducat per Tax Income in \plague Areas (r. down)\\ \hline

Reviewer: Plague Event talks about rounding down Tax Income gained. Here the statement is about the amount to be deducted from Tax Income, so it must be rounded up.

For example, when you have 1 Town in an Area with Plague, then it provides `1/2 = 0.5 (round down)-> 0` Tax Income. But when calculating it here, then you have already added it as 1 Tax Income in the first row of this table, and should subtract `1/2 = 0.5 (round up)-> 1` to make it 0. It works out the same way if you have 3, 5 etc Provinces with Plague. 

As rounding up is default, it is not too important to state it here, but could be helpful because the different statement on Plague Event may cause confusion. So let's keep it, just make it say "round up"

\begin{itemize}
	\item Allied Units in Armies ^^^^2192 Available \manpower
	\item \hspacezero\zsavepos{arrowManpowerStart}Update Manpower
	\item Refresh ½ Exhausted Units (min 3, max 6)
	\item Repair Heavy Ships in Port
	\item Refresh Merchants
	\item Add \colonists = \# of Colonial \claims (max 4)

> 	\item \botrule{Discard Bot \colonists if there is no vacant Territory nor own Colonial \claim (p.~3)}

Reviewer: The condition about Colonial Claims was left out on purpose as it has no effect.

In rules it is stated as "if they have no Colonial [Claim], and there are no vacant Territories."
Both parts of this condition must be true to discard Colonists. However, if there are no vacant Territories, then there cannot be any colonial Claims. This effectively means that if there are no vacant Territories then the whole condition is always true, and if there is any vacant Territory then the whole condition is always false. So mentioning Claims is irrelevant

Author: You are right, I was thinking that a Colonial Claim made a Territory not vacant anymore, but that is wrong.

\end{eubox}\end{dynamiccontents*}

\subsubsection*{C. Discard down to 5 Action Cards}
\begin{itemize}

> 	\item \displaycard in play do not count

Reviewer: If specifically pointing out that display cards in play do not count, then someone might wonder whether other cards in play (like leaders and advisors) do count. I guess it would be better to state that max 5 in hand are allowed, perhaps also making "in hand" bold. It should be obvious that display cards in play are not in hand. Not entirely sure about the wording and whether it fits on one line, though

\end{itemize}

\subsubsection*{D. End of Age Routine}
\begin{itemize}
	\item Go to Final Scoring if
	\begin{itemize}
		\item No more Events left, \conj{or}
		\item PR has ≥100 \prestige and lead of ≥20 \prestige, \conj{or}
		\item PR has all \towns and \vassals on the Map

> 	\end{itemize}

Reviewer: Seems like these could be overlooked if they are stated in "End of Age Routine" but the age is not ending. Simplest solution, in my opinion, would be to move them back to final scoring as they were, and just change the wording here, so it would be clearer that "End of Age Routine" should be skipped if Final Scoring is triggered. 

However, the same argument (about overlooking this) can be made if they are stated in Final Scoring as it doesn't seem like some that should be read before the game ends. So perhaps we should instead think a bit out of the box and deviate from the structure provided by the rulebook. For example, adding another bold heading after "End of Age Routing" that says "Check Final Scoring triggers".

This was something I wasn't sure about already when I first summarized it, and in the end decided to do it as similar to the rulebook as possible, thus also keeping the ambiguity about whether "End of Age Routine" should be applied in case Final Scoring is also triggered. So I'm not trying to say that I liked how it was. Just feeling like the current change is not improving it
